translations/ui-inventory/en.json

```diff
@@ -289,11 +288,11 @@
   "publisher": "Publisher",
   "placeOfPublication": "Place of publication",
   "publisherRole": "Publisher role",
-  "publisherDate": "Publisher date",
+  "publisherDate": "Publication date",
```

Comment:
Can this be consolidated with the `dateOfPublication` key a few lines below this one?